.gitlab-ci.yml

```diff
@@ -94,6 +99,7 @@ build-windows:
 publish-docker:
   stage:                           publish
   only:                            *releaseable_branches
+  <<:                              *no_git
```

Comment:
I guess you need to get the `Dockerfile` somehow.
https://github.com/paritytech/substrate/blob/kubernetes/.gitlab-ci.yml#L143

Reply:
decided to stay with fetch strategies for a while.